Automattic/ghactivity
# Add "Trigger Label Rescan" button into wp-admin settings

This PR adds "Trigger Label Rescan" button into GHActivity settings page. This button should be used to set/update all the label information for every recorded GitHub issue. 

This button will be helpful since right now all the label meta is stored only for new events. 

How to test:
- Locate any GitHub issue which doesn't have / have an outdated labels
- Trigger rescan button
- Wait...
- (check wp logs, it should include some process logs)
- wait a bit more (until you will see "label_scan_action DONE!" message in logs)
- Check updated labels in your GitHub issue
